compiler/rustc_error_codes/src/error_codes/E0463.md

```diff
@@ -11,3 +11,25 @@ extern crate cake_is_a_lie; // error: can't find crate for `cake_is_a_lie`
 You need to link your code to the relevant crate in order to be able to use it
 (through Cargo or the `-L` option of rustc example). Plugins are crates as
 well, and you link to them the same way.
+
+## Common causes
+
+- The crate is not present at all. If using Cargo, add it to `[dependencies]`
+  in Cargo.toml.
+- The crate is present, but under a different name. If using Cargo, look for
+  `package = ` under `[dependencies]` in Cargo.toml.
+
+## Common causes for missing `std` or `core`
+
+- You are cross-compiling for a target which doesn't have `std` prepackaged.
+  Consider one of the following:
+  + Adding a pre-compiled version of std with `rustup target add`
+  + Building std from source with `cargo build -Z build-std`
+  + Using `#![no_std]` at the crate root, so you won't need `std` in the first
+    place. If the error is about `core`, add `#![feature(no_core)]` and
+    `#![no_core]`.
```

Comment:
lol, not sure we should be advocating that... I think talking about `no_core` is likely causing more confusion than it resolves. I don't think `no_core` will ever get stabilized, and until we have a plan for stabilizing it we should probably consider this an internal compiler detail.

Reply:
Ok, I removed the reference to no_core.